sass/components/Table/editable-table-actions.scss

```diff
@@ -0,0 +1,3 @@
+.kubevirt-editable-table-actions {
```

Comment:
maybe `.kubevirt-editable-table__actions` and move to `sass/components/Table/editable-table.scss`